### ISSUE TITLE
Empty Details List Fails Fast-Pass for ARIA attributes must conform to valid values (div[role="grid"])

### ISSUE THREAD

**Issue by taparikh** [OfficeDev/office-ui-fabric-react#11084]
### Environment Information

- **Package version(s)**: 7.8.2
- **Browser and OS versions**:  Chrome Version 77.0.3865.120 (Official Build) (64-bit)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
Automated Checks (Fast pass) fails when there is no rows in the details list. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/tanvi-parikh/pen/xxxWrgw

#### Actual behavior:
When the fastpass is run on an empty detailslist an error is recorded in the Accessibility Insights tool - aria-valid-attr-value which means ARIA attributes must conform to valid values. We don't have a way to change this attribute in the office fabric UI component. 

#### Expected behavior:
This error should not appear

### Documentation describing expected behavior
Here are the failure Details :
Title: WCAG 4.1.1,WCAG 4.1.2: ARIA attributes must conform to valid values (div[role="grid"])
Tags: Accessibility, WCAG 4.1.1, WCAG 4.1.2, aria-valid-attr-value

Issue: ARIA attributes must conform to valid values (aria-valid-attr-value - https://dequeuniversity.com/rules/axe/3.3/aria-valid-attr-value?application=msftAI)

Target application: Catalog Utility Tag Content - https://dev-dxpweb-appservice.azurewebsites.net/

Element path: div[role="grid"]

Snippet: <div role="grid" aria-label="Tag Content table" aria-rowcount="NaN" aria-colcount="7" aria-readonly="true">

How to fix: 
Fix all of the following:
  Invalid ARIA attribute value: aria-rowcount="NaN"

Environment: Chrome version 78.0.3904.87

====

This accessibility issue was found using Accessibility Insights for Web 2.10.3 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Here is a side-by-side screenshot showing the accessibility bug and the component in question 

![acc1](https://user-images.githubusercontent.com/51679982/68242284-1b1c7200-ffc5-11e9-89b3-3c6b6bbaf26f.JPG)

**Comment by JasonGore** [comment]
I tried your CodePen ([changing it to use Fabric 7.55](https://codepen.io/jasongore/pen/yLLKPOG)) and FastPass is not showing any failures. However, the CodePen doesn't seem to be rendering any output so I'm not sure if it's entirely valid.

I've also tried blanking out DetailsList Basic example on the Fabric website and I'm also not seeing any FastPass failures related to DetailsList.

The screenshot you've attached doesn't appear to be from the Fabric repo. The Fabric repo itself won't apply `NaN` to `aria-rowcount`. I suspect code outside of Fabric is passing in an invalid `aria-rowcount` and is the cause of the issue here.

**Comment by JasonGore** [comment]
`DetailsList` calculates `aria-rowcount` internally:
```tsx
const rowCount = (isHeaderVisible ? 1 : 0) + GetGroupCount(groups) + (items ? items.length : 0);
```

It's possible some other prop is causing `NaN`. In this case, maybe `items` is not an array? I'm not sure how the other values could cause `NaN`. However, this is all speculation without a repro.

**Comment by JasonGore** [comment]
An [empty DetailsList example](https://codepen.io/jasongore/pen/pooVoKY) shows no Accessibility Insight errors.

**Comment by qazaahir** [comment]
Looks like the type of empty list was not correct. Changing the type to object to array fixed the error
